modules/tile-converter/src/types.ts

```diff
@@ -1,6 +1,26 @@
+import {Quaternion} from '@math.gl/core';
 import type {GLTFMaterial} from '@loaders.gl/gltf';
+import {Matrix4, Vector3} from 'math.gl';
+import {Tileset3D} from '@loaders.gl/tiles';
+import TileHeader from '@loaders.gl/tiles/src/tileset/tile-3d';
+import {TypedArray} from '@loaders.gl/loader-utils/src/types';
 
-// Types are based on I3S specification https://github.com/Esri/i3s-spec/tree/master/docs/1.7
+export type BoundingVolumes = {
```

Comment:
Q: Should these types eventually go into the `i3s` module? It seems that module also needs them. Could be done later, in another PR.

Reply:
Will move in separate PR.